packages/env/lib/build-docker-compose-config.js

```diff
@@ -32,7 +32,7 @@ function getMounts( config, wordpressDefault = 'wordpress' ) {
 
 	const pluginMounts = config.pluginSources.map(
 		( source ) =>
-			`${ source.path }:/var/www/html/wp-content/plugins/${ source.basename }`
+			`${ source.path }:/var/www/html/wp-content/plugins/${ source.basename }:delegated`
```

Comment:
According to the docs:

> For directories mounted with `delegated` the container’s view of the file system is authoritative, and writes performed by containers may not be immediately reflected on the host file system

It seems like the primary use case for mounting a plugin is for a write on the host file system to reflect immediately on the container, not the other way around. Could you help me understand what `delegated` accomplishes for us?

Reply:
> a write on the host file system to reflect immediately on the container

This is exactly why we use volumes in wp-env. To ensure changes done locally are synced with the container quickly. Basically using this flag optimizes for this use-case while deoptimizing the other direction (container -> host) which we don't really need much.

If you read the post above, it says that's it's approximatively 3 times faster this way (host -> container)